api/src/main/java/io/opentelemetry/trace/propagation/HttpTraceContext.java

```diff
@@ -211,6 +211,9 @@ private static SpanContext extractContextFromTraceParent(String traceparent) {
       if (!VALID_VERSIONS.contains(version)) {
         return SpanContext.getInvalid();
       }
+      if (version.equals("00") && traceparent.length() > TRACEPARENT_HEADER_SIZE) {
```

Comment:
What do you think about making "00" a constant in the class, rather than hard-coding it down here?

Reply:
done